packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -30,6 +30,7 @@ const selectors = {
 	// Publish panel (including post-publish)
 	publishPanel: '.editor-post-publish-panel',
 	viewButton: '.editor-post-publish-panel a:has-text("View")',
+	addNewButton: '.editor-post-publish-panel a[href*="post-new.php"]',
```

Comment:
I'd prefer to see more text selectors used instead. This can be selected as:
`a:text("Add New Post")`.